sktime/transformations/series/summarize.py

```diff
@@ -17,7 +17,7 @@
 
 
 class WindowSummarizer(BaseTransformer):
-    """Transformer for extracting time series features.
+    r"""Transformer for extracting time series features.
```

Comment:
I'm probably missing something, but where exactly do we need the raw string?

Is it for the URL link? If it's only for that, you can take a look at `statsforecast` docstrings.

Reply:
see explanation below